pdns/dnsdist.cc

```diff
@@ -1039,6 +1045,10 @@ bool processResponse(LocalStateHolder<vector<DNSDistResponseRuleAction> >& local
       case DNSResponseAction::Action::HeaderModify:
         return true;
         break;
+      case DNSResponseAction::Action::ServFail:
+        dr.dh->rcode = RCode::ServFail;
+        return true;
+        break;
```

Comment:
I wonder if we should increment `g_stats.ruleServFail` here too?

Reply:
We don't increment any of the ruleXXX counters for ResponseActions right now, and I think that would be confusing because the sums will not match up.

Reply:
Alright, that makes sense :)